packages/ts/react-crud/src/autogrid.tsx

```diff
@@ -317,16 +323,36 @@ export function AutoGrid<TItem>({
 
   useEffect(() => {
     // Update the filtering, whenever the filter changes
-    const grid = ref.current;
+    const grid = gridRef.current;
     if (grid) {
       dataProviderFilter.current = experimentalFilter ?? internalFilter;
       grid.clearCache();
     }
-  }, [experimentalFilter, internalFilter, refreshTrigger]);
+  }, [experimentalFilter, internalFilter]);
 
   return (
-    <Grid itemIdPath={modelInfo.idProperty?.name} {...gridProps} ref={ref}>
+    <Grid itemIdPath={modelInfo.idProperty?.name} {...gridProps} ref={gridRef}>
       {children}
     </Grid>
   );
 }
+
+type AutoGridSignature = <TItem>(
```

Comment:
We could name the type `AutoGrid`, being a type only it would not conflict with the constant below. While potentially confusing for us, it could be beneficial for users if the type they are dealing with is called `AutoGrid` rather than `AutoGridSignature`.

Reply:
Done.